lib/travis/api/app.rb

```diff
@@ -106,6 +106,11 @@ def initialize
         use Travis::Api::App::Middleware::Skylight
         use(Rack::Config) { |env| env['metriks.request.start'] ||= Time.now.utc }
 
+        if ENV['HONEYCOMB_ENABLED_FOR_DYNOS']&.split(' ')&.include?(ENV['DYNO'])
+          Travis.logger.info 'honeycomb enabled'
+          use Travis::Api::App::Middleware::Honeycomb, writekey: ENV['HONEYCOMB_WRITEKEY'], dataset: ENV['HONEYCOMB_DATASET']
```

Comment:
Is adding another layer of middleware going to impact app performance? Do you have any concerns about having multiple monitoring services turned on at once? Do you think there could be any conflicts between services; like skylight artificially impacting the performance that honeycomb reports?